flutter/lib/src/native/java/android_replay_recorder.dart

```diff
@@ -1,40 +1,65 @@
+import 'dart:async';
+import 'dart:isolate';
+
+import 'package:jni/jni.dart';
 import 'package:meta/meta.dart';
 
 import '../../../sentry_flutter.dart';
 import '../../replay/scheduled_recorder.dart';
+import '../../replay/scheduled_recorder_config.dart';
 import '../../screenshot/screenshot.dart';
-import '../sentry_safe_method_channel.dart';
+import 'binding.dart' as native;
 
+// Note, this is currently not unit-tested because mocking the JNI calls is
+// cumbersome, see https://github.com/dart-lang/native/issues/1794
```

Comment:
Please let me know what your ideal mocking solution would look like in the issue.